### DISCUSSION TITLE
v2.3.0-A

### DISCUSSION THREAD

**Discussion by gianni-rosato** [gianni-rosato/svt-av1-psy#113] [Announcements]
Hello everyone! Merry Christmas to those who celebrate – we have a special gift for you this holiday season in the form of **SVT-AV1-PSY v2.3.0-A**! 🎁

Also, this is going to be my last release working heavily on features for SVT-AV1-PSY, at least for a while. You can read more at [my blog post](https://giannirosato.com/blog/post/end-of-psy/), but it has been great fun and I'll miss working on features for this project very much. I'll still be acting as the project's maintainer for now, but very infrequently. Luckily, this project has always been a team effort, and I'm more than certain my talented team members will keep these gears turning without me, so keep tuning in for new releases!

### Features
- Introduced `--enable-tf 2` which adaptively scales temporal filtering strength based on 64x64 block error
- Introduced `--psy-rd`, a feature designed to help retain visual energy to improve perceptual fidelity

### Quality & Performance
- New default `--sharpness` value of 1
- Allowed use of additional interpolation levels for sub pixel motion estimation to improve quality at preset -1
- Variance Boost now samples across three octiles for more consistent boosting behavior
- Increased sharpness on keyframes relative to user-configured `--sharpness` value when using Tune 3

### Bug Fixes
- Fixed a bug preventing use of `--fast-decode 2`
- Fixed crashing with Tune 3 at high CRF values (#96)
- Fixed a bug where CRF and MBR values weren't printing correctly to the terminal

## Support Us

As SVT-AV1-PSY's codebase has become more complex and the encoder's capabilities have increased dramatically, our efforts have scaled in kind. We have poured hours into coding, testing, distributing, and supporting this piece of open-source software entirely for free, and our work isn't stopping any time soon.

If you appreciate the work that we do and you'd like to support us, we are always excited to see code contributions from outside of the core development team. Otherwise, you can support us monetarily via the links below.

**Julio Barba**: *Coming Soon*
**BlueSwordM**: *Coming Soon*
**Clybius**: *Coming Soon*
**Gianni Rosato**: [Donate](https://github.com/sponsors/gianni-rosato?o=esc)

You can also visit our website at [svt-av1-psy.com](https://svt-av1-psy.com/). Any support you can offer goes a long way, and we sincerely appreciate it. ♥️

## Binaries

> *Coming Soon*

*Thanks for using SVT-AV1-PSY!* <3

**Full Changelog**: https://github.com/gianni-rosato/svt-av1-psy/commits/v2.3.0-A

<hr /><em>This discussion was created from the release <a href='https://github.com/gianni-rosato/svt-av1-psy/releases/tag/v2.3.0-A'>v2.3.0-A</a>.</em>

**Comment by gitoss** [comment]
Thanks y'all for your effort on the -psy fork - I hope someone will manage some pull requests to mainline so the features will not be orphaned by further branching development on the 3.0+ mainline.

**Comment by igowrek** [comment]
Thank you for your work!
What's the motivation behind `--sharpness 1` by default? Will that impact performance?

**Reply by gianni-rosato** [reply]
Hi! Defaults are often informed by both metrics and visuals, and we saw both improve with a default`--sharpness` value of 1.
